tests/functional/test_e2e.py

```diff
@@ -75,12 +75,18 @@ def login_as(live_server, page, username):
     expect(page.locator("body")).to_contain_text(f"Logged in as: {username}")
 
 
-def tree_element_is_selected(page, locator):
+def assert_tree_element_is_not_selected(page, locator):
     # We need to wait for a tiny amount to ensure the js that swaps the
-    # selected classes around has run. An arbitrary 20ms seems to be enough.
+    # selected classes around has had time to run before we assert that the
+    # class isn't present. An arbitrary 20ms seems to be enough.
     page.wait_for_timeout(20)
-    classes = locator.get_attribute("class")
-    return "selected" in classes
+    expect(locator).not_to_have_class(re.compile("selected"))
+
+
+def assert_tree_element_is_selected(locator):
```

Comment:
Not sure why these checks started failing on this PR, and only in the docker tests, but they should use the playwright `expect` anyway, which will let them rely on the default wait